package.json

```diff
@@ -43,6 +60,7 @@
     "plotly"
   ],
   "dependencies": {
+    "ajv": "^6.4.0",
```

Comment:
what is this?

Reply:
@etpinard Adding this dependency resolved this error: https://github.com/epoberezkin/ajv/issues/708. If you like to learn more, you might start at https://github.com/npm/npm/issues/19877. I admit I have only cursorily glanced at it. More importantly, I got the above AJV error, and I fixed it by adding this explicit dependency (it was presumably already an implicit dependency).

Hope that explains things. Let me know if you have additional questions or thoughts about it.

Reply:
@JamesCropcho but we don't use `ajv-keyword`, do we?

Reply:
> I forgot about Étienne's comment: we should remove `ajv` from the deps.

Okay, I have done so,